backend/api/Policies/HasPermissionAttribute.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Dal.Security;
+
+namespace Pims.Api.Policies
+{
+    /// <summary>
+    /// HasPermissionAttribute class, provides an attribute that will authorize a users claims to determine if they have permission to perform the requested action.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
+    public class HasPermissionAttribute : TypeFilterAttribute
```

Comment:
Provides a way to decorate our controllers and endpoints to authorize users claims.